### PR TITLE
Add template for the .github 

### PR DESCRIPTION
**Pull request by Madhu-1**
Adding templates required for pull request and issues and also added a template for dependabot

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+# Describe the bug #
+
+A clear and concise description of what the bug is.
+
+# Environment details #
+
+- Image/version of ceph-csi-operator :
+- Image/version of Ceph-CSI driver :
+- Helm chart version :
```
Comment (nb-ohad) [inline .github/ISSUE_TEMPLATE/bug_report.md:15]: This assumes a helm chart which is not yet part of the project. Should we add this after we introduce helm charts?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
+---
+
+# Describe the feature you'd like to have #
+
+What new functionality do you want?
+
```
Comment (nb-ohad) [inline .github/ISSUE_TEMPLATE/feature_request.md:10]:
```suggestion
# Who is the end user and what is the use case where this feature will be valuable?
  
  Who is going to use this feature and under what circumstances?
  
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  guidelines in the [developer
+  guide](https://github.com/ceph/ceph-csi-operator/blob/devel/docs/development-guide.md#commit-messages).
+* [ ] Reviewed the developer guide on [Submitting a Pull
+  Request](https://github.com/ceph/ceph-csi-operator/blob/devel/docs/development-guide.md#development-workflow)
+* [ ] [Pending release
+  notes](https://github.com/ceph/ceph-csi-operator/blob/devel/PendingReleaseNotes.md)
+  updated with breaking and/or notable changes for the next major release.
+* [ ] Documentation has been updated, if necessary.
+* [ ] Unit tests have been added, if necessary.
+* [ ] Integration tests have been added, if necessary.
```
Comment (nb-ohad) [inline .github/pull_request_template.md:49]: Do we really need this as part of the PR message? can't we have this list in the guidelines?

**Review by nb-ohad — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (Madhu-1) [reply .github/ISSUE_TEMPLATE/bug_report.md:15]: Sure

File: .github/pull_request_template.md (hunk 3 above)
Comment (Madhu-1) [reply .github/pull_request_template.md:49]: yes, IMO  no harm in keeping it. it helps us pull request creators to know that these are the basic things required for a PR, most of the time PR creators won't review the guidelines.

Comment (nb-ohad) [reply .github/pull_request_template.md:49]: Ok

**Review by nb-ohad — APPROVED** [review]

**Pull request merged**